DotnetViewComponents/ViewComponents/ImageViewComponent.cs

```diff
@@ -0,0 +1,21 @@
+namespace DotnetViewComponents.ViewComponents
+{
+    using Microsoft.AspNetCore.Mvc;
+    using DotnetViewComponents.ViewModels;
+
+    public class ImageViewComponent : ViewComponent
+    {
+        public IViewComponentResult Invoke(
+            string src,
+            string alt,
+            string caption,
+            string? sizes = null,
+            string? srcSet = null,
```

Comment:
Again, this is pretty obscure, is there a reason we are including it outside of additional attributes?